airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/kotlin/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.kt

```diff
@@ -1319,7 +1319,7 @@ abstract class BaseSqlGeneratorIntegrationTest<DestinationState : MinimumDestina
         strings = ["$", "\${", "\${\${", "\${foo}", "\"", "'", "`", ".", "$$", "\\", "{", "}"]
     )
     @Throws(Exception::class)
-    fun noCrashOnSpecialCharacters(specialChars: String) {
+    open fun noCrashOnSpecialCharacters(specialChars: String) {
```

Comment:
I think you can unopen this?

Reply:
nope, it's overwritten (and @Disabled) in [BigQuerySqlGeneratorIntegrationTest]([url](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java#L428))